ops/instrument_httpclient.go

```diff
@@ -0,0 +1,145 @@
+package ops
+
+import (
+	"context"
+	"time"
+
+	"github.com/gofiber/fiber/v2/utils"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/valyala/fasthttp"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/unit"
+	"go.opentelemetry.io/otel/semconv/v1.17.0"
+
+	"github.com/valkyrie-fnd/valkyrie/internal"
+	"github.com/valkyrie-fnd/valkyrie/rest"
+)
+
+func init() {
+	rest.Pipeline.Register(httpTracingHandler(), httpLoggingHandler(), httpMetricHandler())
+}
+
+func httpTracingHandler() internal.Handler[rest.PipelinePayload] {
```

Comment:
Yes and no depending on how you see it 😄 I intentionally wanted to get rid of otel-specific stuff in in the rest package, since I wanted the rest packate to be as independent as possible and not depend on otel to work. Thats why I tried to reverse the dependency and have `ops` package depend on `rest` instead.